### PR TITLE
Modified templates to match new UI

### PR DESCRIPTION
**Pull request by vtaranenko**

### CODE REVIEW

File: latest/template_all_features.yaml [hunk 1]
```diff
@@ -7,10 +7,10 @@ Parameters:
   SplunkAccessToken:
     Type: String
     NoEcho: true
-    Description: "Copy your Splunk Observability access token from the Observability Cloud AWS integration wizard. You can also find it in Profile > Organization Settings > Access Tokens."
+    Description: "Copy your Splunk Observability access token having INGEST authorization scope from Settings > Access Tokens."
```
Comment (bgola-signalfx) [inline latest/template_all_features.yaml:10]: "having INGEST" --> "with INGEST"

File: latest/template_all_features_regional.yaml [hunk 2]
```diff
@@ -8,10 +8,10 @@ Parameters:
   SplunkAccessToken:
     Type: String
     NoEcho: true
-    Description: "Copy your Splunk Observability access token from the Observability Cloud AWS integration wizard. You can also find it in Profile > Organization Settings > Access Tokens."
+    Description: "Copy your Splunk Observability access token having INGEST authorization scope from Settings > Access Tokens."
```
Comment (bgola-signalfx) [inline latest/template_all_features_regional.yaml:11]: as above

File: latest/template_all_features.yaml (hunk 1 above)
Comment (vtaranenko) [reply latest/template_all_features.yaml:10]: no, "with" is "copy with", having is having

**Review by bgola-signalfx — APPROVED** [review]

**Review by bgola-signalfx — APPROVED** [review]

**Review by aurbiztondo-splunk — APPROVED** [review]

**Comment by bgola-signalfx** [comment]
Resolved in #19

**Pull request closed without merging**